contracts/ERC5018ForBlob.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IERC5018ForBlob.sol";
+
+interface EthStorageContract {
+    function putBlob(bytes32 key, uint256 blobIdx, uint256 length) external payable;
+
+    function get(bytes32 key, uint256 off, uint256 len) external view returns (bytes memory);
+
+    function remove(bytes32 key) external;
+
+    function size(bytes32 key) external view returns (uint256);
+
+    function hash(bytes32 key) external view returns (bytes24);
+
+    function upfrontPayment() external view returns (uint256);
+}
+
+contract ERC5018ForBlob is IERC5018ForBlob {
+
+    address public owner;
+    EthStorageContract public storageContract;
```

Comment:
I think we can implement a simple StorageContract which store data in slots, so that we can test all the function in ERC5018ForBlob.sol